extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -56,4 +57,9 @@ public static function getRelatedVideoData( $title ) {
 	public static function isRelatedVideoEmbedded( $title ) {
 		return !empty( static::getRelatedVideoData( $title ) );
 	}
+
+
+	private static function isArticlePage() {
```

Comment:
maybe `isArticleView`?
Should it be a part of class that can be used in other places? Maybe `Tititle` or something?